drop-transfer/src/ws/client/mod.rs

```diff
@@ -179,7 +224,7 @@ async fn establish_ws_conn(
         Ok(sock) => sock,
         Err(err) => {
             debug!(logger, "Failed to connect: {:?}", err,);
-            return WsConnection::Recoverable;
+            return WsConnection::Stopped;
```

Comment:
Why do we introduce another error variant here? Isn't it just a `Recoverable` error?

Reply:
It was actually present before. When debugging I restored it back and didn't look back at it

Reply:
this was removed